### PR TITLE
Enable XNNPACK in CMake builds

### PR DESCRIPTION
**Pull request by phlash**
enables use of XNNPACK delegate by default on CPU, which provides ~2x performance improvement.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,8 +1,12 @@
 cmake_minimum_required(VERSION 3.16)
-project(deepbacksub CXX)
+project(backscrub CXX)
```
Comment (BenBE) [inline CMakeLists.txt:2]: The name change should be its own separate commit.

**Review by BenBE — APPROVED** [review]
Code-wise LGTM. Didn't run/test locally yet.

File: CMakeLists.txt (hunk 1 above)
Comment (phlash) [reply CMakeLists.txt:2]: Good point, I'll put that elsewhere..
